src/Apps/WorksForYou/Components/MarketingHeader.tsx

```diff
@@ -14,10 +14,12 @@ import {
 } from "@artsy/palette"
 
 const VIDEO_URL = `${sd.FORCE_CLOUDFRONT_URL}/videos/9172018-bn-banner-xl.mp4`
+const COLLECT_URL =
+  "https://artsy.net/collect?split_test[new_collect_page]=new&acquireable=true"
```

Comment:
We still need to figure out the proper way to handle ENV vars across the force / reaction divide. Right now we go through a few hoops. That said were you thinking https://github.com/artsy/force/blob/master/.env.oss#L13? I can follow up with another PR so that this is configurable.